src/StringHelper.php

```diff
@@ -430,4 +430,16 @@ public static function htmlspecialchars(string $string, int $flags, string $enco
             ? htmlspecialchars($string, $flags)
             : htmlspecialchars($string, $flags, $encoding ?: ini_get('default_charset'), $double_encode);
     }
+
+    /**
+     * Truncate the string from the beginning
+     *
+     * @param string $string string to process
+     * @param int $length total of character to truncate
+     * @return string
+     */
+    public static function truncateBegin(string $string, int $length): string
```

Comment:
Add a `, string $suffix = '…'` parameter like we have in the other truncate functions. Also would be good to move the function near to the other truncate functions.

Reply:
@cebe great idea :) 

I updated the PR